README.md

```diff
@@ -55,6 +55,12 @@ For installing weave on other platforms, follow the [integration guides](http://
 Weave respects the environment variable `DOCKER_HOST`, so you can run
 it locally to control a weave network on a remote host.
 
+Weave will periodically check with our servers to see if a new version is available. To disable this, run:
```

Comment:
We should say that new versions are announced in the log.

We really want them to show up in `weave status` too, but can leave that for later.